IFC4x3/Properties/f/FeederLineType_35eHZrPkfE3fcLpjm1KsIK/DocProperty.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="FeederLineType_35eHZrPkfE3fcLpjm1KsIK" Name="FeederLineType" UniqueId="c5a118f5-66ea-4e0e-9995-cedc01536494" PropertyType="p_enumeratedvalue" PrimaryDataType="IfcLabel">
+	<Enumeration xsi:type="DocPropertyEnumeration" xsi:nil="true" href="PEnum_FeederLineType" />
+</DocProperty>
+
```

Comment:
A property of something else (the feeder line)?

Reply:
I have erased it! Commit : c8245c45a7f488800e3e5c5dd32f498fffa28154